content/push/index.textile

```diff
@@ -78,4 +80,34 @@ Each push target device is associated with a unique @deviceId@ and authenticatio
 
 * By employing a standard "Ably key":/auth/basic or "Token":/auth/token, with a @deviceIdentityToken@, a credential generated during registration to assert the device's identity, included in the request header.
 
-The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
\ No newline at end of file
+The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
+
+h4(#Error). Error handling
+
+The @[meta]log:push@ channel logs push notification "metadata":/metadata-stats/metadata. This dedicated channel allows users to monitor system events and errors. It helps track the system's state, debug issues, and ensure that important events are logged and can be reviewed. Metachannels, such as @[meta]log:push@, publish events not otherwise available to clients, providing deeper insights into the system's operations.
+
+The following example subscribes to the @[meta]log:push@ channel:
+
+```[javascript]
+const channel = ably.channels.get('[meta]log:push');
+```
+
+An example scenario for receiving error logs: If Ably encounters an error when delivering a webhook to your endpoint, it sends an event to @[meta]log:push@ with details about the error.
```

Comment:
This webhook example is something that would be returned to `[meta]log`, not `[meta]log:push`.